python/sglang/srt/managers/scheduler.py

```diff
@@ -1493,7 +1363,7 @@ def stream_output(
 
             # Send to detokenizer
             if rids:
-                self.send_to_detokenizer.send_pyobj(
+                self.on_generation_output(
```

Comment:
Shall we add an error message if `self.on_generation_output` is `None`?

Reply:
It seems Scheduler class is an internal implementation detail and not user-facing API, so I personally feel both adding and not adding is OK.